components/converter/Form.js

```diff
@@ -25,12 +25,15 @@ import Anchor from '../Anchor'
 
 import question from './assets/question.svg'
 
-const large = css => breakpoint('large', css)
-const options = ['ANT', 'DAI', 'ETH', 'USDC']
+const ERROR_MIN_ANJ = Symbol('ERROR_MIN_ANJ')
+const ERROR_INSUFFICIENT_BALANCE = Symbol('ERROR_INSUFFICIENT_BALANCE')
 const ANJ_MIN_REQUIRED = bigNum(10)
   .pow(18)
   .mul(10000)
 
+const large = css => breakpoint('large', css)
```

Comment:
Not necessarily something to do now, but we should try to replace the `breakpoint()` calls by `useViewport()`.

Reply:
Let's schedule this for later to reduce the noise on this PR!